websites/A/AniWatch/presence.ts

```diff
@@ -0,0 +1,263 @@
+					label: "Go to Page",
+					url: href,
+				},
+			];
+		}
+	} else if (pathname.startsWith("/az-list")) {
+		presenceData.details = "Looking at Anime list";
+		if (pathname !== "/az-list") {
+			presenceData.state = `Titles starting with ${
+				pathname.substring(9) === "other"
+					? "Other characters"
+					: `${pathname.substring(9)}`
+			}`;
+		}
+		presenceData.smallImageKey = Assets.Search;
+	} else if (pathname.startsWith("/watch2gether/")) {
+		if (pathname === "/watch2gether/")
+			presenceData.details = "Looking for anime rooms";
+		else {
+			const filmName =
+					document.querySelector<HTMLHeadingElement>("h2.film-name"),
+				thumbnail = document.querySelector<HTMLImageElement>(
+					".anis-watch-detail > .anis-content > .anisc-poster > .film-poster > img.film-poster-img"
+				)?.src;
+
+			presenceData.largeImageKey = thumbnail;
+			presenceData.details = "In a room";
+			if (filmName) presenceData.state = `Watching ${filmName.textContent}`;
+			if (data && !data.paused) {
+				[, presenceData.endTimestamp] = presence.getTimestamps(
+					data.currTime,
+					data.duration
+				);
+			}
+			if (buttons) {
+				presenceData.buttons = [
+					{
+						label: "Join Room",
+						url: href,
+					},
+				];
+			}
+		}
+	} else if (pathname.startsWith("/watch")) {
+		const title = document.querySelector<HTMLDataListElement>(
+				"li.breadcrumb-item.dynamic-name.active"
+			),
+			episode = document.querySelector<HTMLSpanElement>(
+				"a.ep-item.active div.ssli-order"
+			),
+			thumbnail = document.querySelector<HTMLImageElement>(
+				".anis-watch-detail > .anis-content > .anisc-poster > .film-poster > img.film-poster-img"
+			)?.src;
+
+		presenceData.largeImageKey = thumbnail;
+		if (title) presenceData.details = title.textContent;
+		if (episode) presenceData.state = `Episode ${episode.textContent}`;
+		if (data && !data.paused) {
+			[, presenceData.endTimestamp] = presence.getTimestamps(
+				data.currTime,
+				data.duration
+			);
+			presenceData.smallImageKey = Assets.Play;
+		} else if (data) presenceData.smallImageKey = Assets.Pause;
+
+		if (buttons) {
+			presenceData.buttons = [
+				{
+					label: "Watch Episode",
+					url: href,
+				},
+			];
+		}
+	} else if (pathname.startsWith("/event/")) {
+		const title = document.querySelector<HTMLDivElement>("div.title"),
+			description = document.querySelector<HTMLDivElement>("div.description");
+		if (title) presenceData.details = `Event: ${title.textContent}`;
+		if (description) presenceData.state = description.textContent;
+	} else if (pathname.startsWith("/community")) {
+		presenceData.details = "Community Post";
+		presenceData.state = document.title;
+		presenceData.smallImageKey = Assets.Question;
+	} else {
+		switch (pathname) {
+			case "/search": {
+				presenceData.details = "Searching";
+
+				const query = document.querySelector(
+					"#main-content > section > div.block_area-header.block_area-header-tabs > div.float-left.bah-heading.mr-4 > h2 > i"
```

Comment:
document queries like these may be too specific. Try removing some of the `> div...` parts in the middle if possible.

Reply:
Doesn't work without the exact string, I've tried.